scripts/newinstall.sh

```diff
@@ -45,6 +45,8 @@ help_flag=false
 noop_flag=false
 
 # Use system python to bootstrap unless otherwise specified
+# This is used to install and run EUPS and will not necessarily
+# be the python in the path being used to build the stack itself.
 PYTHON="${PYTHON:-/usr/bin/python}"
```

Comment:
Now that I come to make the change, I think we do need two distinct logic checks (python in path versus $PYTHON) because it is still the case that if you do not install `miniconda2` you will be building the stack with your PATH python. I'm happy to change the default for `$PYTHON` as it probably makes more sense to use the path version regardless.